README.md

```diff
@@ -8,141 +8,213 @@
   [Official instructions](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15)
-  1. You may experience difficulties if you have brew installed in your home directory. If this is the case option B may work better.
-  2. `brew tap microsoft/mssql-release https://github.com/Microsoft/homebrew-mssql-release`
-  3. `brew update`
-  4. `HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql17 mssql-tools unixodbc`
+  You may experience difficulties if you have brew installed in your home directory. If this is the case option B may work better.
+
+        brew tap microsoft/mssql-release https://github.com/Microsoft/homebrew-mssql-release
+        brew update
+        HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql17 mssql-tools unixodbc
 - (Optional) To start a Sqlserver container in local:
-  1. `docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=MyS3cr3tPassw0rd" -p 1433:1433 --name sqlserver -h sql1 -d mcr.microsoft.com/mssql/server:2019-latest`
+
+        docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=MyS3cr3tPassw0rd" -p 1433:1433 --name sqlserver \
+        -h sql1 -d mcr.microsoft.com/mssql/server:2019-latest
 
 - [Installing MongoDB](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/)
 
 ## Running
 
-Create a `.env` file with the following contents (or use `.env.example` - rename to `.env`):
-
-    - `FLASK_APP=lighthouse`
-    - `FLASK_ENV=development`
-    - `EVE_SETTINGS=development.py`
-
-Option A (in local):
+### Locally using pipenv
 
 1. Enter the python virtual environment using:
 
         pipenv shell
 
-2. Run the app using:
+1. Run the app using:
 
         flask run
 
-**NB:** When adding or changing environmental variables, remember to exit and re-enter the virtual
-environment.
+**NB:** When adding or changing environmental variables, remember to exit and re-enter the virtual environment.
 
-Option B (in Docker):
+### Using Docker
 
 1. Build the docker image using:
 
-        docker build -t lighthouse:develop .
+        docker build --tag lighthouse:develop .
+
+1. Start the services specified in the `docker-compose.yml`:
 
-2. Sql server
+        docker compose up -d
 
-        docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=MyS3cr3tPassw0rd" \
-        -p 1433:1433 --name sql1 -h sql1 \
-        -d mcr.microsoft.com/mssql/server:2019-latest
+    Or, you can start each individully using the instructions in the compose file.
 
-3. Ensure your .env file contains the line
+1. Ensure your `.env` file contains the line
 
         LOCALHOST=host.docker.internal
 
-3. Start the docker container and open a bash session in it with:
+1. Start the docker container and open a bash session in it with:
 
-        docker run --env-file .env -p 5000:5000 -v `pwd`:/code -it lighthouse:develop bash
+        docker run --env-file .env -p 5000:5000 -v $(pwd):/code -it lighthouse:develop bash
```

Comment:
The change in the ENTRYPOINT in docker file means it will change this:
```suggestion
        docker run --env-file .env -p 5000:5000 -v $(pwd):/code -it --entrypoint bash lighthouse:develop
```

Reply:
Yes, good spot, thanks.